_overview/fiat-permissions-overview.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: Permissions in Spinnaker
+order: 210
+published: true
+---
+
+## Overview
+{:.no_toc}
+
+Fiat is the microservice in Spinnaker responsible for authorization (authz) for the other Spinnaker services. By default, it is not enabled, so users are able to perform any action in Spinnaker. This page describes how Fiat interacts with the following Spinnaker services: 
+
+* Clouddriver for account permission
+* Front50 for application permissions
+* Igor for build services permissions
+
+When Fiat is enabled, users start with no permissions and must be explicitly granted permissions. 
+
+For a deeper dive into how authz works for Spinnaker, see [Authorization](https://www.spinnaker.io/setup/security/authorization). Much of Spinnaker's configuration is done through your Halconfig, which can be found in `~/.hal/config`. 
+
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+## Requirements
+
+To use Fiat, you need an external identity provider. Create the user roles and maintain them in the identity provider. Fiat controls what permissions are mapped to roles.  
+
+Fiat supports the following identity providers:
+* SAML groups (includes OAuth ONLY with OIDC)
+* LDAP
+* GitHub teams
+* Google Groups
+  
+In all these methods, users are referenced by a userId, which is determined by the authentication method of your choice.
+
+You also need access to the Halyard machine in order to configure Spinnaker.
+
+## Clouddriver accounts
+
+Clouddriver is the Spinnaker service that interacts with the various cloud providers. When Fiat is enabled, account permissions for Clouddriver determine whether a role or group can perform the following actions:
```

Comment:
So I'm not sure if we want to say `various cloud providers` or just `various providers` since things like docker registries show up under clouddriver also.